docker-hub/official_images.md

```diff
@@ -57,38 +41,67 @@ optimized code. An advanced user could build a custom image with just the
 necessary pre-compiled libraries to save space.
 
 A number of language stacks such as
-[python](https://hub.docker.com/_/python/) and
-[ruby](https://hub.docker.com/_/ruby/) have `-slim` tag variants
+[Python](https://hub.docker.com/_/python/) and
+[Ruby](https://hub.docker.com/_/ruby/) have `-slim` tag variants
 designed to fill the need for optimization. Even when these "slim" variants are
-insufficient, it is still recommended to inherit from an Official Image
+insufficient, it's still recommended to inherit from an Official Image
 base OS image to leverage the ongoing maintenance work, rather than duplicating
 these efforts.
 
-## Submitting Feedback for Docker Official Images
+## Submitting feedback for Docker Official Images
 
 All Docker Official Images contain a **User Feedback** section in their
 documentation which covers the details for that specific repository. In most
 cases, the GitHub repository which contains the Dockerfiles for an Official
 Repository also has an active issue tracker. General feedback and support
 questions should be directed to `#docker-library` on [Libera.Chat IRC](https://libera.chat).
 
-## Creating a Docker Official Image
+## For content publishers
 
-From a high level, an Official Image starts out as a proposal in the form
-of a set of GitHub pull requests. Detailed and objective proposal
-requirements are documented in the following GitHub repositories:
+Docker, Inc. sponsors a dedicated team that's responsible for reviewing and
+publishing all content in Docker Official Images. This team works in
+collaboration with upstream software maintainers, security experts, and the
+broader Docker community.
+
+While it's preferable to have upstream software authors maintaining their
+Docker Official Images, this isn't a strict requirement. Creating
+and maintaining images for Docker Official Images is a collaborative process. It takes
+place openly on GitHub where participation is encouraged. Anyone can provide
+feedback, contribute code, suggest process changes, or even propose a new
+Official Image.
 
-* [docker-library/official-images](https://github.com/docker-library/official-images){:target="_blank" rel="noopener" class="_"}
+> **Note**
+>
+> Docker Official Images are an intellectual property of Docker.
+
+### Vulnerability scanning
+
+[Docker Scout](/scout/){:
+target="blank" rel="noopener" class=""} provides automatic vulnerability scanning 
+for DVP images published to Docker Hub.
+Scanning images ensures that the published content is secure, and proves to
+developers that they can trust the image.
+
+You can enable scanning on a per-repository
+basis, refer to [vulnerability scanning](/docker-hub/vulnerability-scanning/){:
+target="blank" rel="noopener" class=""} for more information about how to use
+it.
```

Comment:
This isn't something end-users can enable on official images -- should this be reworded a little bit to make it clear that users can enable this for _their own_ repositories if they want to?